tools/gestenrekorder/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "gestenrekorder",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development",
+    "test": "ng test",
+    "serve:ssr:gestenrekorder": "node dist/gestenrekorder/server/server.mjs"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "^17.2.0",
+    "@angular/common": "^17.2.0",
+    "@angular/compiler": "^17.2.0",
+    "@angular/core": "^17.2.0",
+    "@angular/forms": "^17.2.0",
+    "@angular/platform-browser": "^17.2.0",
+    "@angular/platform-browser-dynamic": "^17.2.0",
+    "@angular/platform-server": "^17.2.0",
+    "@angular/router": "^17.2.0",
+    "@angular/ssr": "^17.2.2",
```

Comment:
remove duplicate dependencies (from workspace package.json)
(and add necessary angular deps to root package.json)